torch/csrc/distributed/c10d/ProcessGroupNCCL.cpp

```diff
@@ -591,6 +593,17 @@ ProcessGroupNCCL::ProcessGroupNCCL(
             << "\nUSE_HIGH_PRIORITY_STREAM: "
             << options_->is_high_priority_stream
             << "\nNCCL_DEBUG: " << ncclDebugLevel;
+
+
+  static std::once_flag initialize_ucc_lib_flag;
+  std::call_once(initialize_ucc_lib_flag, [&]{
+    ucc_lib_ = loadTorchUCC();
+    if (ucc_lib_ != nullptr) {
+      LOG(INFO) << "[Rank " << rank_  << "] torch_ucc.so loaded";
+    } else {
+      LOG(INFO) << "[Rank " << rank_  << "] torch_ucc.so failed to load";
+    }
```

Comment:
As an early integration, let's keep silent when torch_ucc.so is not used since all the current use cases will hit this else. Otherwise users could be confused.

Reply:
fixed